changelog/1.17.4.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "1.17.4"
+description: "Released on 04/13/21"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.17.4.
+
+### Features ✨
+
+There are no new features in 1.17.4.
+
+### Bug fixes 🐛
+
+- helm: Fixed a malformed envproxy access url value. This fix supersedes the
```

Comment:
Would make the most sense to use code formatting with the literal helm chart value.

I believe `coder.envproxy.accessURL` - is that correct?